src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs

```diff
@@ -198,7 +198,7 @@ private static string GetPlatformName(PlatformTarget platform, bool isSolution)
             }
         }
 
-        private static string GetVerbosityName(Verbosity verbosity)
+        internal static string GetVerbosityName(Verbosity verbosity)
```

Comment:
Should this be moved elsewhere? Feels kind of hacky to call `MSBuildRunner.GetVerbosityName(...)`?

Reply:
Could it be an extension method for `Verbosity` instead?

Reply:
Agreed, perhaps just an extension method on Verbosity?

Reply:
Yep will fix that 😎

Reply:
@mholo65 Addressed & rebased.